tests/functional/dashboard/test_review.py

```diff
@@ -1,4 +1,4 @@
-from datetime import timedelta
+from datetime import timedelta, timezone as datetime_timezone
```

Comment:
please do
```from datetime.timezone import utc```

Reply:
```suggestion
from datetime import timedelta
from datetime.timezone import utc
```

Reply:
okay whatever this is in tests anyway so it does not really matter.